### ISSUE TITLE
some Google apps don't work while ZME is active

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti-tunnel-android#185]
Examples:

* Google Photos upload/backup
* Google News will not refresh

I suspect this is related to Android's ability to detect whether unmetered WiFi is currently available, and that this detection is somehow foiled when ZME is active.

**Comment by qrkourier** [comment]
* Mattermost push notifications don't arrive until I disable ZME, but I need ZME to reach the Mattermost server. This could be because it's using the Google push notification service.

**Comment by qrkourier** [comment]
I've continued to experience frequent problems of this variety with ZME for Android. Workarounds including toggling the VPN, forcibly closing ZME, and rebooting the mobile device. 

Symptoms are still Google provided services cease functioning while the Ziti VPN is active. This appears to impact receiving texts on Fi because I was missing some messages that were sent earlier from another device until I disabled the Ziti VPN.
